pulp_smash/tests/pulp2/python/api_v2/test_sync_publish.py

```diff
@@ -128,8 +128,10 @@ def test_01_first_repo(self):
         * `Pulp #135 <https://pulp.plan.io/issues/135>`_
         * `Pulp Smash #494 <https://github.com/PulpQE/pulp-smash/issues/494>`_
         """
-        if (self.cfg.pulp_version < Version('2.13') or
-                not selectors.bug_is_fixed(135, self.cfg.pulp_version)):
+        if (self.cfg.pulp_version < Version('2.17') or
+                not selectors.bug_is_fixed(3578, self.cfg.pulp_version)):
+            self.skipTest('https://pulp.plan.io/issues/3578')
```

Comment:
@Ichimonji10 shouldn't this `3578` be mentioned in the docstring in the same way the other issues are being mentioned?